### PR TITLE
fix: Increase back arrow click area (#121)

### PR DESCRIPTION
**Pull request by vijaykarthiktk**
increased back arrow click area in appbar  (#121 )

### CODE REVIEW

File: lib/presentation/molecules/top_bar_molecule.dart [hunk 1]
```diff
@@ -82,19 +82,19 @@ class TopBarMolecule extends StatelessWidget {
             const SizedBox(height: 15),
             Row(
               children: [
-                GestureDetector(
-                  onTap: () {
+                IconButton(
```
Comment (guyluz11) [inline lib/presentation/molecules/top_bar_molecule.dart:85]: Should use IconButton from ButtonAtom

File: lib/presentation/molecules/top_bar_molecule.dart [hunk 2]
```diff
@@ -82,19 +82,19 @@ class TopBarMolecule extends StatelessWidget {
...
                     Navigator.of(context).pop();
                   },
-                  child: Icon(
+                  icon: Icon(
                     topBarType == TopBarType.close
                         ? Icons.close_rounded
                         : Icons.arrow_back_ios_rounded,
                     color: iconColor,
-                    size: textTheme.titleMedium!.fontSize,
+                    size: textTheme.titleMedium!.decorationThickness,
```
Comment (guyluz11) [inline lib/presentation/molecules/top_bar_molecule.dart:97]: This should be the same size as the text, please change it to `textTheme.titleLarge.fontSize`

**Review by guyluz11 — CHANGES_REQUESTED** [review]

File: lib/presentation/molecules/top_bar_molecule.dart (hunk 2 above)
Comment (vijaykarthiktk) [reply lib/presentation/molecules/top_bar_molecule.dart]: updated.

File: lib/presentation/molecules/top_bar_molecule.dart (hunk 1 above)
Comment (vijaykarthiktk) [reply lib/presentation/molecules/top_bar_molecule.dart]: sure.

**Review by guyluz11 — CHANGES_REQUESTED** [review]
Remove `color: iconColor,` from top_bar_molecule.dart as it isn't being in use anymore

**Review by guyluz11 — APPROVED** [review]

**Pull request merged**